pyte/screens.py

```diff
@@ -201,7 +197,29 @@ def render(line):
                     yield char
                     next(it)  # Skip stub.
 
-        return ["".join(render(line)) for line in self.buffer]
+        return ["".join(render(self.get_list_of_line(line))) for line in range(self.lines)]
+
+    def get_list_of_line(self, line):
```

Comment:
I'll extend screen.tolist() method with named argument for single lines.

It can be usefull not only for webterm, and buffer must be private thing.